bgpd/bgp_table.h

```diff
@@ -313,4 +313,63 @@ static inline uint64_t bgp_table_version(struct bgp_table *table)
 void bgp_table_range_lookup(const struct bgp_table *table, struct prefix *p,
 			    uint8_t maxlen, struct list *matches);
 
+
```

Comment:
I like the abstraction introduced here, but think the calls should be symmetric/consistent, i.e., both the set and get/from_ calls should be in the form node|info|node_info -- I don't have a strong opinion on which.

Reply:
fixedapotamus